### PR TITLE
Add hint: how to run without KVM

### PR DESCRIPTION
**Pull request by DavHau**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -165,6 +165,15 @@ You can further configure the default mount settings:
...
 
+## Without KVM
+In many cloud environments KVM is not available and therefore nixos-shell will fail with:  
+`CPU model 'host' requires KVM`.  
+This problem can be circumvented by fully emulating the CPU:
+```bash
+export QEMU_OPTS="-cpu Skylake-Client"
+nixos-shell
+```
+
```
Comment (Mic92) [inline README.md:176]:
```suggestion

A full list of supported qemu cpus can be obtained by running `qemu-system-arm -cpu help`.
```

File: README.md [hunk 2]
```diff
@@ -165,6 +165,16 @@ You can further configure the default mount settings:
...
+A full list of supported qemu cpus can be obtained by running `qemu-system-arm -cpu help`.
```
Comment (Mic92) [inline README.md:177]:
```suggestion
A full list of supported qemu cpus can be obtained by running `qemu-system-arm -cpu help`.

```

File: README.md [hunk 3]
```diff
@@ -165,6 +165,17 @@ You can further configure the default mount settings:
...
```
Comment (Mic92) [inline README.md:177]:
```suggestion
A full list of supported qemu cpus can be obtained by running `qemu-system-arm -cpu help`.
```

File: README.md [hunk 4]
```diff
@@ -165,6 +165,17 @@ You can further configure the default mount settings:
 
 Available cache modes are documented in the [9p kernel module].
 
+## Without KVM
```
Comment (Mic92) [inline README.md:168]:
```suggestion
## Without KVM

```

**Comment by Mic92** [comment]
Maybe we don't need this anymore with https://github.com/NixOS/nixpkgs/pull/95956

**Comment by Mic92** [comment]
I am merging this anyhow because old releases are not having it.

**Comment by DavHau** [comment]
> Maybe we don't need this anymore with [NixOS/nixpkgs#95956](https://github.com/NixOS/nixpkgs/pull/95956)

Nice one. I didn't know about the `max` option. We should probably adapt the readme here to use it too. I assume it will increase the performance of the VM.

**Pull request merged**